### ISSUE TITLE
Sudden drop in performance for training and validation

### ISSUE THREAD

**Issue by enggrey** [keras-team/keras#9015]
Have any of you encountered a phenomenon like this before in deep learning? (see image below)
It's almost as though it restarted training.

What could cause such a behavior? How common is it? I know that the training had already reached its peak and would then only overfit thereon, but I still would just expect the validation performance to get worse, not the training also.

I'm training a CNNLSTM with the adam optimizer. I also use dropout with rate 0.3.

![image](https://user-images.githubusercontent.com/7710032/34710793-f4a0abaa-f51c-11e7-865e-1f3a45cf198c.png)

**Comment by dankal444** [comment]
I have multiple spikes like these below almost every training now. 
My intuition tells me it is sth with BatchNormalization, maybe too small batch size (32).
Colleague of mine had similar problems, investigated it without success, I will try too but I don't promise anything.
![image](https://user-images.githubusercontent.com/24862989/36354745-56e55082-14d9-11e8-87fb-9b6dc17628a9.png)
